tools/python/util/ort_format_model/operator_type_usage_processors.py

```diff
@@ -24,7 +24,8 @@ def _ort_constant_for_domain(domain: str):
     # This list is limited to just the domains we have processors for
     domain_to_constant_map = {'ai.onnx': 'kOnnxDomain',
                               'ai.onnx.ml': 'kMLDomain',
-                              'com.microsoft': 'kMSDomain'}
+                              'com.microsoft': 'kMSDomain',
+                              'org.pytorch.aten': 'kPytorchATenDomain'}
```

Comment:
does it need to be added here?
see comment above: `# This list is limited to just the domains we have processors for`

Reply:
TBH, I am not sure what the author meant with "processor" in this context :)

Reply:
I think it doesn't need to be added.
@skottmckay can you confirm?

Reply:
@skottmckay could you confirm whether adding `ort.pytorch.aten` domain to `operator_type_usage_processors.py` is required for replacing ORT's custom Aten OP by the recently introduced PyTorch's?

Reply:
This is used for determining the operators and types used by a model for creating a build with a reduced set of supported operators/types. I would not have expected pytorch ATen operators to be applicable to that scenario given the ATen library, IIRC, is massive, saving a few MB for the ORT build would be insignificant.

Reply:
Sounds like we have the answer @thiagocrepaldi and @edgchen1

Reply:
@skottmckay ATEN library is not shipped in ORT nor ORT training wheel packages. Users are required to install pytorch as a dependency for training, which has the aten library there. So keeping or not this code would not imply any binary size increase. In that case, we could keep this line

However, if tools/python/util/ort_format_model/operator_type_usage_processors.py is meant for a ORT package that does not include ORTModule, then we could remove it

@skottmckay @edgchen1 What I do? Remove it or keep it?

Reply:
What would happen in the scenario where there are aten ops in the model? 
- and we do keep this line?
- and we don't keep this line?

Reply:
AFAIK only ORT training supports aten ops

Reply:
Inferencing definitely does NOT support aten ops.

This script is used to create the config for a reduced ops/types build. The context for wanting to do that is you need a smaller binary size for deployment to say a mobile device. I don't think that will ever be applicable to a training build that includes ATEN support.

It doesn't technically hurt to keep the line, but I think it's slightly better not to have it as you'll get an error if you're attempting to create a config based on a model with ATEN ops. We have never attempted or thought through the implications of doing a reduced ops build with that (or the sanity of even wanting to), so at least an error will alert you to someone trying to do that.